config/locales/hyrax.en.yml

```diff
@@ -10,12 +10,14 @@ en:
           file_format_sim: File Format
           format_sim: Format
           generic_type_sim: Generic Type
+          has_model_ssim: Browse
```

Comment:
This seems odd.  Maybe it gets you what you want and doesn't cause any ill effects ... but it seems odd to me for 'Browse' to be set as the name of the 'has_model' facet.  Not necessarily saying it's wrong ... just seems odd.